test/verifiedTests/analysis/testfindMIIS/testfindMIIS.m

```diff
@@ -0,0 +1,53 @@
+% The COBRAToolbox: testfindMIIS.m
+%
+% Purpose:
+%     - testfindMIIS tests the debugging tool findMIIS that takes as input
+%     an infeasible model and returns to  Minimal Irreducible Infeasible
+%     Submodel
+%
+% Author:
+%     - Marouen BEN GUEBILA 02/12/2017
+
+global CBTDIR
+global ILOG_CPLEX_PATH
+
+addpath(genpath(ILOG_CPLEX_PATH));
+
+%Test presence of required toolboxes.
+v = ver;
+bioPres = any(strcmp('Bioinformatics Toolbox', {v.Name})) && license('test','bioinformatics_toolbox');
```

Comment:
Are these toolboxes really required for the test? 
From the code it looks like it should only depend on the CPLEX solver.
You could actually write an assert(changeCobraSolver('ibm_cplex') == 1), but I'm not sure, whether the solver is installed on all CI systems.
If not, the current test will pass without it actually having tested anything.
@laurentheirendt ?

Reply:
Why not do:
```Matlab
solverOK = changeCobraSolver('ibm_cplex');
if solverOK
% ...
end
```

Reply:
Ok, so ibm_cplex is installed on all CI systems? 
I just was not sure about it.

Reply:
`changeCobraSolver` solver takes care of that.

Reply:
If these toolboxes are required, then this check should be in `findMIIS`, not in the test only.

Reply:
@tpfau cplex is not installed on windows and mac machines, thus the `if solverOK`

Reply:
ok. 
I just wasn'T sure, and if it ends up not being tested at all that would be inconvenient :)